Procfile

```diff
@@ -5,7 +5,9 @@ worker_buildapi_pending: newrelic-admin run-program celery -A treeherder worker
 worker_buildapi_running: newrelic-admin run-program celery -A treeherder worker -Q buildapi_running --maxtasksperchild=20 --concurrency=5
 worker_buildapi_4hr: newrelic-admin run-program celery -A treeherder worker -Q buildapi_4hr --maxtasksperchild=20 --concurrency=1
 worker_store_pulse_jobs: newrelic-admin run-program celery -A treeherder worker -Q store_pulse_jobs --maxtasksperchild=20 --concurrency=1
+worker_store_pulse_resultsets: newrelic-admin run-program celery -A treeherder worker -Q store_pulse_resultsets --maxtasksperchild=20 --concurrency=1
```

Comment:
Sure, let's bump them to 3 for now and adjust as needed.  The traffic on the resultsets will be dramatically lower than for jobs, of course.  So I don't know if it matters much either way.  We don't even fall behind for jobs, afaict.